packages/zoe/src/zoe.js

```diff
@@ -514,6 +517,9 @@ const makeZoe = (additionalEndowments = {}) => {
           return removePurse(issuerRecord);
         }),
 
+      updatePublicAPI: newPublicAPI =>
+        instanceTable.update(instanceHandle, { publicAPI: newPublicAPI }),
```

Comment:
If you buy the argument above, please error here if it is already set.

Reply:
Ok, that sounds good to me, thanks